packages/mui-system/src/Container/createContainer.tsx

```diff
@@ -79,7 +79,6 @@ export default function createContainer<Theme extends RequiredThemeStructure = D
         marginLeft: 'auto',
         boxSizing: 'border-box',
         marginRight: 'auto',
-        display: 'block', // Fix IE11 layout when used with main.
```

Comment:
This was introduced in https://github.com/mui/material-ui/pull/19971 but there's no description of the bug it was causing.